src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/HashAlgorithm.cs

```diff
@@ -92,6 +92,11 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        public void Clear() 
+        {
+            (this as IDisposable).Dispose();
```

Comment:
It's what [Desktop did](http://index/?query=hashalgorithm&rightProject=mscorlib&file=system%5Csecurity%5Ccryptography%5Chashalgorithm.cs&line=206) although perhaps for no particular reason.